package.json

```diff
@@ -43,14 +36,13 @@
     "rollup": "^0.50.0",
     "rollup-plugin-buble": "^0.16.0",
     "rollup-watch": "^4.3.1",
-    "rxjs": "^5.2.0",
+    "rxjs": "^6.0.0",
+    "rxjs-compat": "^6.0.0",
```

Comment:
Probably don't want this included by default, leave that to consumers of the library to decide whether they need it or not.

Reply:
These are dev deps